weave

```diff
@@ -471,11 +471,8 @@ case "$COMMAND" in
         fi
         if [ "$1" = "-password" ] ; then
             [ $# -gt 1 ] || usage
-            PASSWORD=$2
-            shift 2
-        elif [ "$1" = "-passwordfile" ] ; then
-            [ $# -gt 1 ] || usage
-            PASSWORDFILE=$2
+            WEAVE_PASSWORD="$2"
+            export WEAVE_PASSWORD
```

Comment:
exporting env vars from scripts is generally bad. in this particular case it will affect any subsequent `weave launch`.

`-password` should continue to be implemented as before; yes it is less secure than the env var route, but that is why the latter exists.